apisupport/Build/apisupport-LINUX-GETRANDOM.c

```diff
@@ -0,0 +1,16 @@
+#include <sys/random.h>
+
+#include <stdlib.h>
+
+#define BUFLEN 2
+
+int
+main(void)
+{
+	char buf[BUFLEN];
+
+	if (getrandom(buf, BUFLEN, 0) != BUFLEN)
```

Comment:
I don't know if we want to actually check that we can get bytes from `getrandom()`, or just let this auto-succeed and handle the possible failure in the calling function.

Reply:
Oops, I meant to add this comment to `apisupport/apisupport_linux_getrandom.c`, not this file.